server/app/models/catalog.rb

```diff
@@ -50,4 +50,9 @@ def stream_to_protocol(stream)
       request_rate_concurrency:
     )
   end
+
+  def default_cursor_field(stream_name)
+    stream = catalog["streams"].find { |s| s["name"] == stream_name }
```

Comment:
Please avoid naming it s. Instead use stream

Reply:
The stream is singular and the output obtained after filtering the catalog streams using the stream name.